cloudinstall/utils.py

```diff
@@ -101,6 +102,41 @@ def write_status_file(status='', msg=''):
     spew(status_file, json.dumps(dict(status=status, msg=msg)))
 
 
+def populate_config(opts):
+    """ populate configuration suitable for loading in the config
+    object merging in cli options.
+
+    :param opts: argparse Namespace class of options
+    """
+    cfg_cli_opts = vars(opts)
+    cfg = {}
+
+    def sanitize_config_items(_cfg):
+        """ remove false and null items """
+        return {k: v for (k, v) in _cfg.items()
+                if v is not None and v is not False}
```

Comment:
I'm not sure we will always want to remove False args.
If you add a boolean arg in the future and expect it to be able to be True or False, this will strip it unexpectedly.

Is this here because Config.__getattr__ is overloaded to return False if the key doesn't exist?
I'm not sure that's a good default, it's not a common thing to do in Python and if it causes bugs it wouldn't be the first thing someone would look for.